var/sabath/assets/sabath/models/c/cosmoflow/requirements.txt

```diff
@@ -0,0 +1,7 @@
+pandas
+wandb
+horovod==0.19.*
+
+# make sure horovod is re-compiled if environment is re-built
```

Comment:
Why building Horovod from source? That's usually the most fragile part of the process

Reply:
See above.